brave/brave-core
# fix(wallet): show transaction dialog for approval/reject

- In AssetDetailsActivity and AccountDetailsActivity
- Show live transaction status change in list
- Refactor Utils


Resolves https://github.com/brave/brave-browser/issues/19969

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
- Verify the expected behaviour in the ticket
- Verify the expected behaviour in Accounts Details List screen (Accounts-> select any account)
- Verity the unapproved status changes in the transaction list
